WorkflowUI/Sources/Hosting/WorkflowHostingController.swift

```diff
@@ -150,4 +155,23 @@ public final class WorkflowHostingController<ScreenType, Output>: UIViewControll
     }
 }
 
+// MARK: Observation Events
+
+public protocol WorkflowHostingControllerEvent: WorkflowUIEvent {
```

Comment:
i'm not sure how useful this protocol is... since you have to know all the generic types up front, there isn't really a good way to do something like: 'given this `WorkflowUIEvent`, conditionally do something if it was emitted from a WorkflowHostingController'. you basically can't know that the event came from a hosting controller unless we were to add either some shared metadata to the base event protocol that specifies that, or make a marker protocol that ignores the associated types (or something else?).

Reply:
Agree, this doesn't seem to add value other than enforcing some naming conventions for the generic types

Reply:
remove this for now, and may revisit in the future